lua/kulala/globals/init.lua

```diff
@@ -10,5 +10,6 @@ M.SCRATCHPAD_ID = "kulala://scratchpad"
 M.HEADERS_FILE = plugin_tmp_dir .. "/headers.txt"
 M.BODY_FILE = plugin_tmp_dir .. "/body.txt"
 M.COOKIES_JAR_FILE = plugin_tmp_dir .. "/cookies.txt"
+M.CONSOLE_FILE = plugin_tmp_dir .. "/console.txt"
```

Comment:
This shouldn't be part of this PR, eh?

Reply:
Sorry, this is part of another PR. https://github.com/mistweaverco/kulala.nvim/pull/189